docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -4,154 +4,203 @@ approvers:
 title: Schedule GPUs
 ---
 
-{% capture overview %}
-
-Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
-This page describes how users can consume GPUs and the current limitations.
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread
+across nodes. The support for NVIDIA GPUs was added in v1.6 and has gone through
+multiple backwards incompatible iterations. This page describes how users can
+consume GPUs across different Kubernetes versions and the current limitations.
+
+## v1.6 and v1.7
```

Comment:
Can we move this section at the end and have the current version be the first one ?
We could add a summary :)

Reply:
It flows better this way. I made the recommendation to use device plugins **bold**.